resources/qml/UserProfile.qml

```diff
@@ -0,0 +1,155 @@
+import QtQuick 2.9
+import QtQuick.Controls 2.3
+import QtQuick.Layouts 1.2
+import QtQuick.Window 2.3
+
+import im.nheko 1.0
+
+import "./device-verification"
+
+ApplicationWindow{
+    property var user_data
+    property var colors: currentActivePalette
+
+    id:userProfileDialog
+    height: 500
+    width: 500
+    modality:Qt.WindowModal
+    Layout.alignment: Qt.AlignHCenter
+    palette: colors
+
+    UserProfileList{
+        id: userProfileList
+        userId: user_data.userId
+        onUserIdChanged : {
+            userProfileList.updateDeviceList()
+        }
+        onDeviceListUpdated : {
+            modelDeviceList.deviceList = userProfileList
+        }
+    }
+
+    Component {
+		id: deviceVerificationDialog
+		DeviceVerification {}
+	}
+    Component{
+        id: deviceVerificationFlow
+        DeviceVerificationFlow {}
+    }
+
+    background: Item{
+        id: userProfileItem
+        width: userProfileDialog.width
+        height: userProfileDialog.height
+        anchors.margins: {
+            top:20
+        }
+
+        ColumnLayout{
+            anchors.fill: userProfileItem
+            width: userProfileDialog.width
+            spacing: 10
+
+            Avatar{
+                id: userProfileAvatar
+                url:chat.model.avatarUrl(user_data.userId).replace("mxc://", "image://MxcImage/")
+                height: 130
+                width: 130
+                displayName: modelData.userName
+                Layout.alignment: Qt.AlignHCenter
+            }
+
+            Label{
+                id: userProfileName
```

Comment:
This should be bigger than the other text I guess, since it is probably the most important aspect of the dialog:

![grafik](https://user-images.githubusercontent.com/16312324/85565207-75367a00-b61e-11ea-89bc-a48b1cb7b628.png)

Reply:
(Just the username and maybe you can also color it in the usercolor? :D)